mixing/mixclient/client.go

```diff
@@ -516,11 +516,11 @@ func (c *Client) prDelay(ctx context.Context, p *peer) error {
 	sendBefore := epoch.Add(-timeoutDuration - maxJitter)
 	sendAfter := epoch.Add(timeoutDuration)
 	var wait time.Duration
-	if now.After(sendBefore) {
+	if now.Equal(sendBefore) || now.After(sendBefore) {
```

Comment:
I'd suggest a single check as follows:

```suggestion
	if !now.Before(sendBefore)
```

Quick verification:

```
	now := time.Now()
	sendBefore := now
	fmt.Printf("sendBefore == now, result1: %v, result2: %v\n", !now.Before(sendBefore), now.Equal(sendBefore) || now.After(sendBefore))
	sendBefore = now.Add(time.Nanosecond)
	fmt.Printf("sendBefore = now + 1 nanosecond, result1: %v, result2: %v\n", !now.Before(sendBefore), now.Equal(sendBefore) || now.After(sendBefore))
	sendBefore = now.Add(-time.Nanosecond)
	fmt.Printf("sendBefore = now - 1 nanosecond, result1: %v, result2: %v\n", !now.Before(sendBefore), now.Equal(sendBefore) || now.After(sendBefore))
```

Output:
```
sendBefore == now, result1: true, result2: true
sendBefore = now + 1 nanosecond, result1: false, result2: false
sendBefore = now - 1 nanosecond, result1: true, result2: true
```